src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
Correct me if I am mistaken as I have not taken a look at it in a while, but, I think this is more or less the same case as jQuery, right? If I don't remember it wrong, it returns an UMD module but also exports itself as a browser's global so plugins can attach themselves to it if needed.